nemo/collections/tts/waveglow_modules.py

```diff
@@ -207,16 +207,15 @@ def denoise(self, audio, strength=0.1):
         audio_denoised = librosa.core.istft(audio_spec_denoised * audio_angles)
         return audio_denoised, audio_spec_denoised
 
-    def forward(self, mel_spectrogram):
+    def forward(self, *args):
```

Comment:
is it possible to put named arguments (if necessary with default values back instead of "*argc")?

Reply:
Done